charts/ndb-operator/templates/apps_v1_deployment_controller-manager.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: {{ include "ndb-operator.fullname" . }}-controller-manager
+  namespace: {{.Release.Namespace}}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      control-plane: controller-manager
```

Comment:
let's use `{{ include "ndb-operator.fullname" . }}-` as prefix of every label value